netci.groovy

```diff
@@ -46,8 +46,15 @@ def osList = ['Ubuntu', 'OSX', 'Windows_NT']
                         batchFile(buildString)
 
                         if (configuration == 'Debug') {
-                            prJobDescription += " + CoreCLR tests"
-                            batchFile("tests\\runtest.cmd /coreclr Top200")
+                            if (isPR) {
+                                prJobDescription += " + CoreCLR tests"
```

Comment:
Since you're touching this: could you replace the `+` with `and` so that we can use "dotnet-bot test Blah" for this? The CI has trouble parsing out the `+`.